wfcGen.py

```diff
@@ -0,0 +1,264 @@
+import random
+
+from PIL import Image
+
+
+res = 3
+cropShift = False
+cells = {}
+cropped = []
+for ins in range(4):
+    field = Image.open("image.png")
+    if cropShift:
+
+        field = field.rotate(90*ins)

```

Comment:
These line breaks are rather confusing IMO. Do you think the line `if cropShift:` is logically closer to `field = Image.open("image.png")` than to `field = field.rotate(90*ins)`?